mkosi/resources/mkosi.md

```diff
@@ -51,17 +53,21 @@ that may generate disk images with a number of bells and whistles.
 The following command line verbs are known:
 
 `summary`
-:   Outputs a human-readable summary of all options used for building an
+:   Output a human-readable summary of all options used for building an
     image. This will parse the command line and `mkosi.conf` file as it
-    would do on `build`, but only output what it is configured for and not
+    would do on `build`, but only print what it is configured for and not
```

Comment:
"what it is" => "what is"

Reply:
"what it is … configured for". It is correct.